src/cli.rs

```diff
@@ -68,6 +71,20 @@ impl Rover {
             .get(RoverEnvKey::Home)?
             .map(|p| PathBuf::from(&p)))
     }
+
+    pub(crate) fn get_git_context(&self) -> GitContext {
+        // use these env vars as overrides to GitContext creation
+        let branch = self.env_store.get(RoverEnvKey::GitBranch).unwrap();
+        let committer = self.env_store.get(RoverEnvKey::GitCommitter).unwrap();
+        let commit = self.env_store.get(RoverEnvKey::GitCommit).unwrap();
+        // this url is still dirty, make sure not to use it anywhere else :)
+        let remote_url = self.env_store.get(RoverEnvKey::GitRemoteUrl).unwrap();
+
+        // constructing GitContext with a set of overrides from env vars
+        let git_context = GitContext::new(branch, committer, commit, remote_url);
+        tracing::debug!("Git Context: {:?}", git_context);
```

Comment:
```suggestion
        tracing::debug!("Git Context", ?git_context);
```

Reply:
This doesn't work for me, I get an error saying 

```error: expected expression, found `?`
  --> src/cli.rs:97:40
   |
97 |         tracing::debug!("Git Context", ?git_context);
   |                                        ^ expected expression
```